### ISSUE TITLE
Provide command to insert missed activity 

### ISSUE THREAD

**Issue by amferraboli** [brendalf/workhours#5]
When it comes to tracking worked hours or any other activity, one of the most recurrent issue people face is forgetting to register the start/end of an activity.  Adding the possibility to insert a past record which was forgotten would remove the burden to have to manually insert it in the log file, improving usability and engagement with the tool.

**Comment by brendalf** [comment]
I agree. Maybe a `wh insert` command that asks for the date and the type of action, WDYT?
